e2e/cypress/integration/custom_status/custom_status_spec.js

```diff
@@ -0,0 +1,557 @@
+
+            // * The custom status should be removed from the Recents
+            cy.get('#custom_status_modal #statusSuggestion__recents').should('not.contain', customStatus.text);
+        });
+
+        it('MM-T3847_4 should set default status when clicked on the status', () => {
+            // # Set a custom status from the Suggestions by clicking on it and then clicking "Set Status" button
+            cy.get('#custom_status_modal .statusSuggestion__content').contains('span', defaultStatus.text).click();
+            cy.get('#custom_status_modal .GenericModal__button.confirm').click();
+
+            // * Check if custom status is successfully set by checking the emoji in the sidebar header
+            cy.get('#headerInfoContent span.emoticon').invoke('attr', 'data-emoticon').should('contain', defaultStatus.emoji);
+        });
+
+        it('MM-T3847_5 should show status set in step 4 in the top in the Recents list', () => {
+            openCustomStatusModal();
+
+            // # Click on the clear button in the custom status modal input
+            cy.get('#custom_status_modal .StatusModal__clear-container').click();
+
+            // * Custom status modal input should be empty
+            cy.get('#custom_status_modal input.form-control').should('have.value', '');
+
+            // * The set status should be present at the top of the Recents list
+            cy.get('#custom_status_modal #statusSuggestion__recents .statusSuggestion__row').first().find('.statusSuggestion__text').should('have.text', defaultStatus.text);
+        });
+
+        it('MM-T3847_6 should remove the default status from Recents and show in the Suggestions', () => {
+            cy.get('#custom_status_modal #statusSuggestion__recents .statusSuggestion__row').first().trigger('mouseover');
+            cy.get('#custom_status_modal #statusSuggestion__recents .statusSuggestion__row').first().get('.suggestion-clear').should('be.visible');
+
+            // * The set status should be present in Recents list and not in the Suggestions list
+            cy.get('#custom_status_modal #statusSuggestion__recents').should('contain', defaultStatus.text);
+            cy.get('#custom_status_modal #statusSuggestion__suggestions').should('not.contain', defaultStatus.text);
+
+            // # Click on the clear button of the topmost suggestion
+            cy.get('#custom_status_modal #statusSuggestion__recents .statusSuggestion__row').first().get('.suggestion-clear').click();
+
+            // * The status should be moved from the Recents list to the Suggestions list
+            cy.get('#custom_status_modal #statusSuggestion__recents').should('not.contain', defaultStatus.text);
+            cy.get('#custom_status_modal #statusSuggestion__suggestions').should('contain', defaultStatus.text);
+        });
+    });
+
+    describe('MM-T3850 Verifying where the custom status appears', () => {
```

Comment:
Please move to its own file `custom_status_5_spec.js`

```suggestion
    describe('Custom Status - Verifying Where Custom Status Appears', () => {
```